array/DNAm/preprocessing/checkRconfigFile.r

```diff
@@ -80,24 +80,9 @@ if (!toupper(arrayType) %in% c("HM450K", "V1", "V2")) {
   warning("\nUnrecognised arrayType. Must be 'HM450K', 'V1' or 'V2'\n")
 }
 
-
-for (i in c("Individual_ID", "Sex")) {
-  if (!i %in% bioVar) {
-    bad_parameter_exists <- TRUE
-    warning("\n'", i, "' must be included in bioVar\n")
-  }
-}
-
-if (!"Cell_Type" %in% bioVar && ctCheck) {
+if (!"Cell_Type" %in% projVar && ctCheck) {

```

Comment:
A possible problem here is that someone's data might have cell type data, but it is in a column called "cell_type" or "cellType" *etc.*

Not related to this PR though. Just wanted to mention it

Reply:
This looks to already be covered by the checkColnamesSampleSheet.r script

```
cond_cols <- c('Sex','Genotype_IID','Cell_Type')

#3. Check conditional column names ---------------------------------------------------------------------------
cat("Sourcing conditional variables from config.r",'\n')
source(configFile)
cond_status <- c(sexCheck,snpCheck,ctCheck) # T/Fs from config file
cat(paste0(c("sexCheck","snpCheck","ctCheck"),"=", cond_status),'\n')

# subset conditional colnames to those TRUE in config
cond_cols.filtered <- cond_cols[cond_status]

if(all(cond_status==F)){
	cat("No conditional variables to check")
}else{
	cat("3. Checking conditional column names: ",'\n')
	cat(cond_cols.filtered,'\n')
	checkColnames(sampleSheet, cond_cols.filtered, type='Conditional')
}
```

Reply:
Fantastic